Volume/Views/View Models/MorePublicationRow.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  PublicationRow.swift
+//  Volume
+//
+//  Created by Cameron Russell on 10/29/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import SwiftUI
+
+/// `MorePublicationsRow`displays the basis information about a publications the user is not currently following
+struct MorePublicationRow: View {
+    
+    var publication: Publication
+    
+    @State private var buttonTapped = false
+    @State private var expanded = false
+    @State private var spacing: CGFloat = 12
+
+    private func determineTruncation(_ geometry: GeometryProxy) {
+        let total = publication.description.boundingRect(
+                with: CGSize(width: geometry.size.width, height: .greatestFiniteMagnitude),
+                options: .usesLineFragmentOrigin,
+                attributes: [.font: UIFont.systemFont(ofSize: 11)],
+                context: nil
+            )
+
+        if total.size.height > geometry.size.height {
+            self.expanded = true
+        }
+    }
```

Comment:
Are you sure this is necessary? I thought we found an almost-solution with List.

Reply:
Long story short, the list ended up moving independently of the entire page (i.e you could scroll to the end of the list and still be at the top of the page). We still changed something because the pseudo-constraints work with all iPhones now.